# Time of sensor data acquisition, and the coordinate frame ID
Header header

# Can message ident 0x19F8021D
string DIRECTION_REFERENCE 
float64 COURSE_OVER_GROUND 
float64 SPEED_OVER_GROUND